lib/carrierwave/storage/postgresql_lo.rb

```diff
@@ -75,12 +49,21 @@ def retrieve!(identifier)
       end
 
       def identifier
-        @oid ||= uploader.model.read_attribute(uploader.mounted_as) || connection.lo_creat
+        @oid ||= uploader.model.read_attribute(uploader.mounted_as) || create_large_object
       end
 
       def connection
         @connection ||= uploader.model.class.connection.raw_connection
       end
+
+      private
+      def create_large_object
```

Comment:
I left this here because it is the only piece of code under the `PostgresqlLo` class that isn't adapter specific. I suppose it could be moved out, but just a single method, I feel it's acceptable.